src/main/java/edu/ucsb/cs156/courses/controllers/CourseOverTimeController.java

```diff
@@ -32,9 +33,37 @@ public class CourseOverTimeController {
     @ApiOperation(value = "Get a list of courses over time")
     @GetMapping(value = "/search", produces = "application/json")
     public ResponseEntity<String> search(
+        @ApiParam(
+            name = "StartQtr",
+            type = "string",
+            value = "starting quarter in yyyyq format, e.g. 20231 for W23, 20232 for S23, etc. (1=Winter, 2=Spring, 3=Summer, 4=Fall)",
+            example = "20231",
+            required = true
+        )
         @RequestParam String startQtr,
+        @ApiParam(
+            name =  "endQtr",
+            type = "String",
+            value = "ending quarter in yyyyq format, e.g. 20231 for W23, 20232 for S23, etc. (1=Winter, 2=Spring, 3=Summer, 4=Fall)",
+            example = "20231",
+            required = true
+        )
         @RequestParam String endQtr,
+        @ApiParam(
+            name =  "subjectArea",
+            type = "String",
+            value = "simplified area name followed with full area name",
+            example = "EARTH - Earth Science",
```

Comment:
This documentation isn't quite right; look at the weird way course IDs are formatted in the database:
![image](https://github.com/ucsb-cs156-s23/proj-courses-s23-7pm-3/assets/77405911/4d687717-70f2-4a5b-82e8-6da025ce1951)
This is the reason why the `findByQuarterRangeAndCourseId` method is called with `makeFormattedCourseId`, so that the input can be fixed to query the database.

I think just `value = "simplified area name"` and `example = EARTH` is more accurate.